### PR TITLE
Allow extra tolerations in buildpod admission test

### PR DESCRIPTION
**Pull request by adambkaplan**
Let build pods have additional tolerations in integration test.
These may come from enabled admission plugins.

### CODE REVIEW

File: test/integration/buildpod_admission_test.go [hunk 1]
```diff
@@ -135,20 +135,28 @@ func TestBuildOverrideTolerations(t *testing.T) {
 		},
 	}
 
+	requiredTolerations := make(map[string]kapi.Toleration)
```
Comment (soltysh) [inline test/integration/buildpod_admission_test.go]: Can you please change `tolerations` to `[]v1.Toleration` up above and build the map there?

Comment (adambkaplan) [reply test/integration/buildpod_admission_test.go]: @soltysh updated

**Comment by adambkaplan** [comment]
ping @soltysh

File: test/integration/buildpod_admission_test.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (soltysh) [inline test/integration/buildpod_admission_test.go]: For future, `corev1` is the prefferred alias for this api.

Comment (adambkaplan) [reply test/integration/buildpod_admission_test.go]: I think VSCode may have done that...

**Pull request merged**